app/views/RoomActionsView/index.js

```diff
@@ -667,117 +391,465 @@ class RoomActionsView extends React.Component {
 		);
 	}
 
-	renderRoomInfo = ({ item }) => {
+	renderRoomInfo = () => {
 		const { room, member } = this.state;
-		const { name, t, topic } = room;
-		const { baseUrl, user, theme } = this.props;
+		const {
+			rid, name, t, topic
+		} = room;
+		const {
+			baseUrl, user, theme, fontScale
+		} = this.props;
 
 		const avatar = RocketChat.getRoomAvatar(room);
+		const isGroupChat = RocketChat.isGroupChat(room);
 
 		return (
-			this.renderTouchableItem((
-				<>
-					<Avatar
-						text={avatar}
-						size={50}
-						style={styles.avatar}
-						type={t}
-						baseUrl={baseUrl}
-						userId={user.id}
-						token={user.token}
-					>
-						{t === 'd' && member._id ? <Status style={sharedStyles.status} id={member._id} /> : null }
-					</Avatar>
-					<View style={[styles.roomTitleContainer, item.disabled && styles.roomTitlePadding]}>
-						{room.t === 'd'
-							? <Text style={[styles.roomTitle, { color: themes[theme].titleText }]} numberOfLines={1}>{room.fname}</Text>
-							: (
-								<View style={styles.roomTitleRow}>
-									<RoomTypeIcon type={room.prid ? 'discussion' : room.t} status={room.visitor?.status} theme={theme} />
-									<Text style={[styles.roomTitle, { color: themes[theme].titleText }]} numberOfLines={1}>{RocketChat.getRoomTitle(room)}</Text>
-								</View>
-							)
+			<List.Section>
+				<List.Separator />
+				<Touch
+					onPress={() => this.onPressTouchable({
+						route: 'RoomInfoView',
+						// forward room only if room isn't joined
+						params: {
+							rid, t, room, member
 						}
-						<Markdown
-							preview
-							msg={t === 'd' ? `@${ name }` : topic}
-							style={[styles.roomDescription, { color: themes[theme].auxiliaryText }]}
-							numberOfLines={1}
-							theme={theme}
-						/>
-						{room.t === 'd' && <Markdown msg={member.statusText} style={[styles.roomDescription, { color: themes[theme].auxiliaryText }]} preview theme={theme} numberOfLines={1} />}
+					})}
+					style={{ backgroundColor: themes[theme].backgroundColor }}
+					accessibilityLabel={I18n.t('Room_Info')}
+					accessibilityTraits='button'
+					enabled={!isGroupChat}
+					testID='room-actions-info'
+					theme={theme}
+				>
+					<View style={[styles.roomInfoContainer, { height: 72 * fontScale }]}>
+						<Avatar
+							text={avatar}
+							size={50 * fontScale}
+							style={styles.avatar}
+							type={t}
+							baseUrl={baseUrl}
+							userId={user.id}
+							token={user.token}
+						>
+							{t === 'd' && member._id ? <Status style={sharedStyles.status} id={member._id} /> : null }
+						</Avatar>
+						<View style={[styles.roomTitleContainer, isGroupChat && styles.roomTitleDisabled]}>
```

Comment:
Let's just remove the chevron when the roomInfo is disabled instead of apply an opacity.